Getting-Started/Data/Data-Types/index.md

```diff
@@ -3,17 +3,17 @@ versionFrom: 8.0.0
 ---
 
 # Data Types
-*A Data Type defines the type of input for a property. So when adding a property (on Document Types, Media Types and Members) when selecting the Type you are selecting a Data Type. There are a number of preconfigured Data Types available in Umbraco and more can be added in the Settings section.*
+*A Data Type defines the type of input for a property. So when adding a property (on Document Types, Media Types and Members) and selecting the Type you are selecting a Data Type. There are a number of preconfigured Data Types available in Umbraco and more can be added in the Settings section.*
 
 ## What is a Data Type?
-A Data Type can be something very (textstring, number, true/false,...) or more complex (multi node tree picker, image cropper, Grid Layout).
+A Data Type can be something very simple (textstring, number, true/false,...) or it can be more complex (multi node tree picker, image cropper, Grid Layout).
```

Comment:
"Simple" is one of the words that we recommend not using 😅 
The word would actually make sense in this particular case though. Perhaps we could use something like "basic" instead? Or "light"? 🤔

Reply:
Ye, I don't like simple either, just couldn't think of a suitable alternative :) basic could work.